client/src/curriculum/scrim.scss

```diff
@@ -0,0 +1,167 @@
+@use "../ui/vars" as *;
+
+.scrim {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  grid-area: scrim;
+  height: 100%;
+  justify-content: center;
+  justify-self: center;
+  max-width: 24rem;
+  width: 100%;
+
+  @media (min-width: $screen-lg) {
+    justify-self: end;
+  }
```

Comment:
I would recommend adding `margin-top: 1rem` to `.scrim` on mobile:

| Before | After |
|--------|--------|
| <img width="545" alt="image" src="https://github.com/mdn/yari/assets/495429/bad6d871-b137-4569-9852-076f647d984a"> | <img width="545" alt="image" src="https://github.com/mdn/yari/assets/495429/307be3c7-90a4-4502-9e27-d571fd23638c"> | 

/cc @AnujaRajput727